apps/taskman/service.py

```diff
@@ -93,12 +93,26 @@ def create_or_update_task(self, flaw: Flaw, fail_if_exists=False) -> Response:
             }
             return Response(data=res, status=409)
 
+        modules = flaw.affects.values_list("ps_module", flat=True).distinct()
+        products = PsProduct.objects.filter(ps_modules__name__in=modules)
+        labels = [f"flawuuid:{str(flaw.uuid)}", f"impact:{flaw.impact}"]
+        for product in products:
+            if product.team:
+                labels.append(f"team:{product.team}")
+        if flaw.is_major_incident:
+            labels.append("major_incident")
+        if flaw.cve_id in flaw.title:
+            summary = flaw.title
+        else:
+            summary = f"{flaw.cve_id} {flaw.title}"
```

Comment:
CVE ID may be blank at the first stages of the flaw life cycle. Will this work with such flaw?

Reply:
Double checked and the answer is no, it will add an extra space in summary, nice catch, will fix.
(`"" in "some string"` is always true)